### DISCUSSION TITLE
HOW CAN I DRAG BAR VALUE 0???

### DISCUSSION THREAD

**Discussion by yete666** [chrispahm/chartjs-plugin-dragdata#88] [Q&A] [answered]
Hello, 
what I said, I can not drag a bar with value 0, how can I do to drag a bar from value 0 to 10 for example.??,
the bar disappears and I can not select it, can you select outside the bar??

**Comment by chrispahm** [comment]
Hey 👋 
Please see the following issue for a few suggestions on how to solve this: https://github.com/chrispahm/chartjs-plugin-dragdata/issues/52

**Comment by chrispahm** [comment]
Hey 👋 
Please see the following issue for a few suggestions on how to solve this: https://github.com/chrispahm/chartjs-plugin-dragdata/issues/52

**Comment by yete666** [comment]
thank you for your quick response!!!

I can't get it to work with Horizontal Bar - Simple Horizontal Bar

    xAxes: {
                    minBarLength: 10,
                    ticks: {
				  reverse: false
                      }
                  }

thank you

**Comment by yete666** [comment, marked as answer]
I already found the problem!!
 updated chart.js, moved from to datasets.

data: {
    datasets: [{
        barPercentage: 0.5,
        barThickness: 6,
        maxBarThickness: 8,
        minBarLength: 2,
        data: [10, 20, 30, 40, 50, 60, 70]
    }]
};


thank you
